src/serlo-editor/plugins/text/static-components/static-math.tsx

```diff
@@ -7,7 +7,10 @@ import 'katex/contrib/mhchem'
 import type { MathElement } from '../types/text-editor'
 import { sanitizeLatex } from '@/schema/sanitize-latex'
 
-export function StaticMath({ src, inline }: MathElement) {
+export type StaticMathProps = Omit<MathElement, 'children'>
+
+/** this is quite big 🐘, if you can load it dynamically  */
```

Comment:
Maybe it's my bad English, but at first I thought this means "if you can load this dynamically, then it is quite big". A comma would have probably saved me a couple of minutes, so like: "This is quite big. If you can, load it dynamically."

Reply:
Sorry for the confusion. I'll just add another elephant to make sure ;)